samples/pytorch/Makefile

```diff
@@ -1,28 +1,47 @@
 TOP = $(abspath ../..)
 include $(TOP)/defs.mak
 
-APPBUILDER=$(TOP)/scripts/appbuilder
+APPBUILDER    = $(TOP)/scripts/appbuilder
+APP_NAME      = python3
+APP_PATH      = myst/bin/$(APP_NAME)
+LAUNCH_WAIT   = 90 # waiting time in seconds for service launch
+SHUTDOWN_WAIT = 2  	# waiting time in seconds before service being terminated
 
 ifdef STRACE
-# OPTS = --strace
+OPTS = --strace
 endif
 
-all: appdir rootfs
+all: appdir build
 
 appdir:
-	$(APPBUILDER) -v Dockerfile
+	$(APPBUILDER) -v -d Dockerfile.rootfs
 
-rootfs: appdir
-	sudo $(MYST) mkext2 appdir ext2rootfs
-	sudo chown $(USER).$(USER) ext2rootfs
-	$(MYST) fssig --roothash ext2rootfs > roothash
-	# $(MYST) mkcpio appdir rootfs
+build: appdir package.pem
+	$(MYST) package-sgx appdir package.pem config.json
```

Comment:
With a huge CPIO rootfs included in the Enclave image in the packaged mode, the Mystikos Enclave load time will be quite long. Can the Makefile also support a mode of EXT2 rootfs?